api/database/factories/ModelFactory.php

```diff
@@ -138,6 +138,16 @@
     ];
 });
 
+$factory->define(App\Models\Image::class, function (\Faker\Generator $faker) {
+    return [
+            'image_id' => $faker->uuid,
+            'public_id' => $faker->unique()->word,
+            'version' => time(),
+            'format' => 'png',
+            'alt' => $faker->slug()
```

Comment:
This would be better defined as `$faker->sentence` as it is a natural language sentence.